old/bindable-input.html

```diff
@@ -0,0 +1,71 @@
+<!--
+    @license
+    Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+    This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+    The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+    The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+    Code distributed by Google as part of the polymer project is also
+    subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../polymer/polymer.html">
+
+<!--
+@element bindable-input
+-->
+
+<dom-module id="bindable-input">
+
+  <style>
+    :host {
+      font: inherit;
+      color: inherit;
+      margin: 0;
+    }
+  </style>
+
+</dom-module>
+
+<script>
+
+(function() {
+
+  Polymer({
+
+    is: 'bindable-input',
+
+    extends: 'input',
```

Comment:
Does `Polymer` skip creating a `shadowRoot` if there is no template provided for the extended element?